test/integration/full/configure-options/configure-options.js

```diff
@@ -1,50 +1,82 @@
-describe('Check Configure Options', function() {
+describe('Configure Options', function() {
```

Comment:
I named it `Check configure options` to distinguish between the other `options-parameter` tests in the same folder. Perhaps these new tests should be combined with those instead? Or all together?

Reply:
oh! I see, you're right. I put them in the wrong place. My bad.

Reply:
Actually, no, this is correct. The `options parameter` is the `axe.run` options. This is for `axe.configure`. Your existing test was for `axe.configure({ check })`, this new one is `axe.configure({ disableOtherRules })`.